NanoVNASaver/Marker/Widget.py

```diff
@@ -351,3 +351,7 @@ def updateLabels(self,
             self.label['s21phase'].setText(format_phase(s21.phase))
             self.label['s21polar'].setText(
                 str(round(abs(s21.z), 2)) + "∠" + format_phase(s21.phase))
+            self.label['s21magshunt'].setText(format_magnitude(abs(s21.shuntImpedance())))
+            self.label['s21magseries'].setText(format_magnitude(abs(s21.seriesImpedance())))
+            self.label['s21realimagshunt'].setText(format_complex_imp(s21.shuntImpedance()))
```

Comment:
The format_complex_imp() will need its optional second argument set to True to properly display negative real numbers here (same holds for the next line)

Reply:
Updated, ammended the commit.